Terminal.Gui/ConsoleDrivers/WindowsDriver.cs

```diff
@@ -1707,20 +1703,22 @@ public override void UpdateCursor ()
 
 		public override void End ()
 		{
+
 			WinConsole.Cleanup ();
 			WinConsole = null;
 
+			// Disable alternative screen buffer.
+			Console.Out.Write ("\x1b[?1049l");
+
 			// Needed for Windows Terminal
 			// Clear the alternative screen buffer from the cursor to the
 			// end of the screen.
 			// Note, [3J causes Windows Terminal to wipe out the entire NON ALTERNATIVE
 			// backbuffer! So we need to use [0J instead.
 			Console.Out.Write ("\x1b[0J");
 
-			// Disable alternative screen buffer.
-			Console.Out.Write ("\x1b[?1047l");
-
-			// Console.Out.Flush () is not needed. See https://stackoverflow.com/a/20450486/297526
+			Console.Out.Write ("\x1b[!p");
+			Console.Out.Write ("\x1b[!p");
```

Comment:
I really don't know what this is for, I'm curious. I already test but I didn't see any difference.

Reply:
I left this in by accident. It's soft-reset. But doesn't actually do anything.